packages/@lwc/perf-benchmarks/src/__benchmarks__/ssr/slots/slot-create-container-5k.benchmark.js

```diff
@@ -0,0 +1,29 @@
+import { renderComponent } from '@lwc/ssr-runtime';
+import SlotUsage from '@lwc/perf-benchmarks-components/dist/ssr/benchmark/slotUsageComponent/slotUsageComponent.js';
+import Store from '@lwc/perf-benchmarks-components/dist/ssr/benchmark/store/store.js';
+
+const SSR_MODE = 'asyncYield';
+const NUMBER_OF_ROWS = 5000;
+
+benchmark(`ssr/slot/shadow/create/5k`, () => {
```

Comment:
We could probably stand to add some light-dom-slot tests as well. Opened an issue: https://github.com/salesforce/lwc/issues/4784